### DISCUSSION TITLE
Docker image for web client?

### DISCUSSION THREAD

**Discussion by hadim** [rustdesk/rustdesk#7591] [General]
Given the ongoing work on the rustdesk web client, I was wondering whether you could make available (nightly and stable) docker images for the web client so it's easy for people to test it out and/or deploy it without having to build it from scratch.

**Comment by Znerox** [comment]
Bump, I would also really like to see this. 

I see there is a pre-built image available, mentioned at the bottom [here](https://rustdesk.com/docs/en/dev/build/web/) (the commands listed there doesn't work, there is no "latest" tag, so you need to add :v1.0.0 at the end if you want to use that). But that image hasn't been updated in close to two years, and I had problems getting it to work. It's also possible to use [this](https://docs.linuxserver.io/images/docker-rustdesk/) image which is the full client, accessible in the browser via VNC, but that has it's own drawbacks.
